### PR TITLE
build(ivy): make sure animation_world prodserver and devserver work as expected

### PR DESCRIPTION
**Pull request by matsko**

### CODE REVIEW

File: packages/core/test/bundling/animation_world/BUILD.bazel [hunk 1]
```diff
@@ -30,8 +45,29 @@ ng_rollup_bundle(
     ],
 )
 
-http_server(
+ts_devserver(
     name = "devserver",
+    entry_module = "@angular/core/test/bundling/animation_world",
```
Comment (gregmagolan) [inline packages/core/test/bundling/animation_world/BUILD.bazel]: nit: if you add
```
    additional_root_paths = [
        "npm/node_modules/tslib",
    ],
```
to ts_devserver then you can just add `"@npm//tslib",` to static_files and load `/tslib.js` without needing the genrule

**Review by gregmagolan — APPROVED** [review]
LGTM. One small comment to simplify the ts_derver only

File: packages/core/test/bundling/animation_world/BUILD.bazel (hunk 1 above)
Comment (gregmagolan) [reply packages/core/test/bundling/animation_world/BUILD.bazel]: Example: https://github.com/bazelbuild/rules_nodejs/blob/master/e2e/ts_devserver/genrule/BUILD.bazel#L47

Comment (matsko) [reply packages/core/test/bundling/animation_world/BUILD.bazel]: Fixed. Thx for the tip.

**Comment by AndrewKushnir** [comment]
@matsko could you plz have a look at the lint errors on CI? Thank you.

**Comment by matsko** [comment]
@AndrewKushnir done

**Pull request closed without merging**